### PR TITLE
Codacy fixes

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Fixing all issues of the categories
* Code Style
* Unused Code

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
👍

**Comment by AndyScherzinger** [comment]
cc @mario for second approval :)

**Comment by AndyScherzinger** [comment]
pinging @mario for second review & merge 😀

**Comment by AndyScherzinger** [comment]
pinging @mario for second review & merge 😀

File: src/main/java/com/owncloud/android/ui/preview/PreviewMediaFragment.java [hunk 1]
```diff
@@ -491,10 +489,10 @@ private void sendShareFile() {
 
     private void prepareVideo() {
         // create helper to get more control on the playback
-        mVideoHelper = new VideoHelper();
-        mVideoPreview.setOnPreparedListener(mVideoHelper);
-        mVideoPreview.setOnCompletionListener(mVideoHelper);
-        mVideoPreview.setOnErrorListener(mVideoHelper);
+        VideoHelper videoHelper = new VideoHelper();
```
Comment (mario) [inline src/main/java/com/owncloud/android/ui/preview/PreviewMediaFragment.java]: A thousand times yes! :D

**Comment by mario** [comment]
Build failed @AndyScherzinger ?

**Comment by AndyScherzinger** [comment]
@mario drone failed (git to be exact), restarted the build, now green again :)

**Comment by AndyScherzinger** [comment]
Build is green again @mario :D

**Comment by mario** [comment]
Approved

**Pull request merged**